openstack/mrs/v1/job/Create.go

```diff
@@ -0,0 +1,128 @@
+	// Starts with / or s3a://. The OBS path does not support files or programs encrypted by KMS.
+	// Spark Script must end with .sql while MapReduce and Spark Jar must end with .jar.sql and jar are case-insensitive.
+	JarPath string `json:"jar_path" required:"true"`
+	// Key parameter for program execution. The parameter is specified by the function of the user's program.
+	// MRS is only responsible for loading the parameter.
+	// The parameter contains a maximum of 2,047 characters, excluding special characters such as ;|&>'<$, and can be left blank.
+	// NOTE:
+	// When entering a parameter containing sensitive information (for example, login password),
+	// you can add an at sign (@) before the parameter name to encrypt the parameter value.
+	// This prevents the sensitive information from being persisted in plaintext.
+	// Therefore, when you view job information on the MRS, sensitive information will be displayed as asterisks (*).
+	// For example, username=admin @password=admin_123.
+	Arguments string `json:"arguments,omitempty"`
+	// Path for inputting data, which must start with / or s3a://. Set this parameter to a correct OBS path.
+	// The OBS path does not support files or programs encrypted by KMS.
+	// The parameter contains a maximum of 1,023 characters, excluding special characters such as ;|&>'<$, and can be left blank.
+	Input string `json:"input,omitempty"`
+	// Path for outputting data, which must start with / or s3a://. A correct OBS path is required.
+	// If the path does not exist, the system automatically creates it.
+	// The parameter contains a maximum of 1,023 characters, excluding special characters such as ;|&>'<$, and can be left blank.
+	Output string `json:"output,omitempty"`
+	// Path for storing job logs that record job running status. The path must start with / or s3a://. A correct OBS path is required.
+	// The parameter contains a maximum of 1,023 characters, excluding special characters such as ;|&>'<$, and can be left blank.
+	JobLog string `json:"job_log,omitempty"`
+	// SQL program path
+	// This parameter is needed by Spark Script and Hive Script jobs only, and must meet the following requirements:
+	// Contains a maximum of 1,023 characters, excluding special characters such as ;|&><'$. The address cannot be empty or full of spaces.
+	// The path must start with / or s3a://. The OBS path does not support files or programs encrypted by KMS.
+	// The path must end with .sql.sql is case-insensitive.
+	HiveScriptPath string `json:"hive_script_path,omitempty"`
+
+	IsProtected bool `json:"is_protected,omitempty"`
+	IsPublic    bool `json:"is_public,omitempty"`
+}
+
+type CreateDistCpOpts struct {
+	// 5: DistCp, importing and exporting data.
+	JobType int `json:"job_type"`
+	// Job name
+	// Contains only 1 to 64 letters, digits, hyphens (-), and underscores (_).
+	// NOTE:
+	// Identical job names are allowed but not recommended.
+	JobName   string `json:"job_name"`
+	ClusterId string `json:"cluster_id"`
```

Comment:
why here ClusterId?
but on the line 27 ClusterID?
and everywhere around the PR
Id or ID?

Reply:
Id